lib/rails/perftest/railties/testing.tasks

```diff
@@ -4,14 +4,25 @@ namespace :test do
   task 'perftest:benchmark_mode' do
     ENV["BENCHMARK_TESTS"] = '1'
   end
+  if Rake::Task.task_defined?('test:prepare')
```

Comment:
I don't think the `db:test:prepare` dependency is necessary. It's executed on demand when `test_help.rb` is loaded.